app/views/home/index.html.erb

```diff
@@ -1 +1,10 @@
+<h1 class="mb-3">Condomínios</h1>
 
+<% @condos.each do |condo| %>
+    <%= link_to condo_path(condo.id), class: 'link-underline link-underline-opacity-0' do %>
+    <div class="bg-warning-subtle py-3 px-5 text-black mb-3 rounded-4">
+        <p class="display-6"><b><%= condo.name %></b></p>
+        <h6>Endereço: <%= condo.full_address %></h6>
```

Comment:
Porque `<h6>`? Isso não me parece um título. Além disso, se for um título, seria mais adequado um `<h2>` para seguir a hierarquia, mas se foi para estilizar, é melhor utilizar as CSS para não fugir do escopo da função do HTML.